### PR TITLE
4089 ledger table change name 'quantity' to 'units'

### LINKED ISSUE #4089: Ledger table: change name 'Quantity' to 'Units'
## What went wrong? 😲

On the ledger tab the column 'quantity' means number of Units.

Without any specification it can be confusing as to whether means 'Packs' or 'Units'.

telegram-cloud-photo-size-1-5132363654342225164-y

## Your environment 🌱

Desktop app v2.1.0-IA-test-2

### PR DESCRIPTION
**Pull request by roxy-dao**
Fixes #4089

# 👩🏻‍💻 What does this PR do?
Changed quantity in ledger to unit quantity... 
Screenshot 2024-09-30 at 8 58 12 AM

The quantity in Returns is being used in the stepper, so I thought it was fine to leave it but idk? 

# 🧪 Testing

- [ ] Go to ledger
- [ ] See `Unit Qty` instead of quantity

# 📃 Documentation

- [ ] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [x] **These areas should be updated or checked**: 
  1. Screenshots for ledger
  2.

### CODE REVIEW

File: client/packages/common/src/intl/locales/en/distribution.json [hunk 1]
```diff
@@ -66,7 +66,6 @@
   "label.outbound-shipment": "Outbound shipment",
   "label.placeholder": "Placeholder",
   "label.delete": "Delete",
-  "label.quantity": "Quantity",
```
Comment (roxy-dao) [inline client/packages/common/src/intl/locales/en/distribution.json:69]: Removed cause it's in common

File: client/packages/system/src/Stock/Components/Ledger/useLedgerColumns.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export enum ColumnKey {
   'DateTime' = 'datetime',
   'Time' = 'time',
   'Name' = 'name',
-  'Quantity' = 'quantity',
+  'UnitQuantity' = 'unitQuantity',
```
Comment (lache-melvin) [inline client/packages/system/src/Stock/Components/Ledger/useLedgerColumns.ts:16]: I think the value still needs to be `quantity` - ledger is showing blank values now! 😁 

Screenshot 2024-09-30 at 11 01 26 AM

**Review by lache-melvin — CHANGES_REQUESTED** [review]
Thanks Roxy, just need to fix the display 😅 

Assuming all the extra translations you just pulled through from develop? I think its good to include them in the release, though seeing them here is reminding me we need that way to keep the RC branch up to date with translations/what to do if we change translation strings between versions...

**Comment by roxy-dao** [comment]
> Thanks Roxy, just need to fix the display 😅
> 
> Assuming all the extra translations you just pulled through from develop? I think its good to include them in the release, though seeing them here is reminding me we need that way to keep the RC branch up to date with translations/what to do if we change translation strings between versions...

The translations used to go to main... but now it's going straight to develop... maybe can change it back to main to get latest translations? 👀

**Comment by lache-melvin** [comment]
> The translations used to go to main... but now it's going straight to develop... maybe can change it back to main to get latest translations? 👀

I think we're sticking with develop, but there's some issues to hash out, #4060, #4673 :)

**Review by lache-melvin — APPROVED** [review]
Mean as TY 🚀

**Comment by mark-prins** [comment]
Two reasons for using `develop` rather than `main`
1. Branch protection on `main` meant that weblate was unable to push to it without creating a PR ( there's a way around this though, which we're doing for develop, so this isn't an issue now )
2. It allows our translators to translate new functionality before release

I wouldn't want to change back to `main` due to the second point - that situation is more common than the need to pull translations into an RC / hotfix branch; which I think is what's needed here?

File: client/packages/common/src/intl/locales/en/common.json [hunk 3]
```diff
@@ -211,7 +211,7 @@
   "label.contact-tracing": "Contact Tracing",
   "label.contact-tracing-type": "Type of contact",
   "label.context": "Context",
-  "label.cost": "Cost",
+  "label.cost": "Pack cost price",
```
Comment (jmbrunskill) [inline client/packages/common/src/intl/locales/en/common.json:214]: Shouldn't we use a more descriptive `label.cost_per_pack` to make translation easier and have less mistakes?
I remember Adam raised something about french translations not having the correct details in them?

Comment (mark-prins) [reply client/packages/common/src/intl/locales/en/common.json:214]: yes! is requested in #4891

**Pull request merged**